meta-koi/recipes-android/brcm-patchram-plus/brcm-patchram-plus/patchram.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
The `brcm-patchram-plus` service might not be needed on `koi` since we can use the bluebinder and Android Bluetooth service combination. Can you remove these?